src/coreclr/src/jit/emit.cpp

```diff
@@ -5691,6 +5696,42 @@ void emitter::emitDataGenEnd()
 #endif
 }
 
+/*****************************************************************************
+ * Generates a data section constant
+ *
+ * Parameters:
+ *     cnsAddr  - memory location containing constant value
+ *     cnsSize  - size of constant in bytes
+ *     cnsAlign - alignment of constant in bytes
+ *
+ * Returns constant number as offset into data section.
+ */
```

Comment:
This is a copy/paste comment that needs to be updated (to the new comment style, also)

Reply:
OK